system.h

```diff
@@ -134,13 +137,14 @@ void system_execute_startup(char *line);
 
 // But We don't need such a big value, and we do need the high bit free
 //#define MAX_LINE_NUMBER 9999999
-#define LINENUMBER_EMPTY_BLOCK 0x8000 //the other bit, used as a flag
-#define LINENUMBER_SPECIAL      0x4000
+#define LINENUMBER_EMPTY_BLOCK 0x10000 //the other bit, used as a flag
+#define LINENUMBER_SPECIAL      0x4000 //denotes Homing and Probing among a couple other processes
+#define LINENUMBER_SPECIAL_SERVO 0x8000 //denotes force servoing
```

Comment:
I'm a little uncomfortable redefining constants like this. Why not make the force servoing bit 0x10000 and leave 0x8000 as the empty block?